### ISSUE TITLE
2.7.0-rc3 pre-release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2557]
- #2526
- #2556
- #2531
- #2544
- #2494
- #2545 
- #2542
- #2563 

## Pick list
```
git cherry-pick f1bf0acb06a281515a3cc2c5cb2b98eae0a18794
git cherry-pick 401a0531581ee5813a5ca922ce8477e8e9611c77
git cherry-pick 9151df479cb7caf5c1973ca37033b3660266a39c
git cherry-pick 31bb70744a464de33ac1a5d832ad289a78fbd25b
git cherry-pick 07b1873789b55daa0111e42526fc63740947fa56
git cherry-pick eec3fbe97ee1e033ac5393c05e6c0702ae3302dc
git cherry-pick f513e2b0ed10e4aeca15fb2ef944e7ca39513bbf
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.7.0-rc2..HEAD --pretty='format:- %s'`
```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._


- Closes #2543 Retire theme-debug lando/ddev tooling. (#2545)
- Closes #2491: Re-exported configuration files for 2.7.x (#2542)
- Closes #2552 Cannot remove instances of recurring events; Update drupal/smart_date requirement from 3.6.1 || 3.7.2 to 4.0.2 (#2494)
- Closes #2300 Enable Search Keywords view filter (#2544)
- Closes #2529: Remove DB updates added before 2.6.0. (#2531)
- Closes #2541 Downgrade az_digital_asset_library back to experimental due to login issues. (#2556)
- Closes #2525 Add error handling to 2.7.x az_news database update. (#2526)
- #2560 Update github action to use specific release branch (2.7.x) of scaffolding repo (#2563)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created:
https://github.com/az-digital/az_quickstart/releases/tag/2.7.0-rc3
